### ISSUE TITLE
Ready screen instantly comes up when choosing a hero.

### ISSUE THREAD

**Issue by Darkmega18** [darklordabc/Legends-of-Dota-Redux#1025]
Immediately when I choose my hero my screen changes to the hero versus ready screen. usually I've already picked my abilities, but it's stopping me from accessing build changer as as soon as I choose a hero it gives me a ready screen I can't close anymore which shouldn't be there even when in the middle of a game and just closes the menu and doesn't let me choose anything anymore. it might have something to do with the last 15 minutes thing you guys just added. I'm using local version on the Dota map.

Short version. Choose hero even when not locked instant gives ready screen with bots, not sure about with people so I can't even choose hero main stat and stuff.

also just found out default mischief on monkey king says ability not learnt when clicking on it just now. :V but thats another bug.

**Comment by darklordabc** [comment]
This is fixed

**Comment by Darkmega18** [comment]
GG!
